google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/FieldValueTest.java

```diff
@@ -50,75 +50,75 @@
 
   @Test
   public void testFromPb() {
-    FieldValue value = FieldValue.fromPb(BOOLEAN_FIELD);
+    FieldValue value = FieldValue.fromPb(BOOLEAN_FIELD, null);
```

Comment:
If it's common to pass `null` as a second parameter, do we want to create a `fromPb` overload that only requires the first parameter?

Reply:
So I was thinking about same thing it during making this change. I'll add the overloaded version.